### PR TITLE
Linters

### PR DESCRIPTION
**Pull request by speedandfunction-anribolon**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{
```
Comment (niedzielski) [inline .eslintrc.json:1]: Should we be using tabs in JSON files?

File: .eslintignore [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+/dist/*.map.json
+
+# These JSON files must be manually linted to allow comments which are more valuable than
+# presentation. eslint-plugin-json errors on overrides and file disables are unsupported:
+# https://github.com/azeemba/eslint-plugin-json/issues/17.
+/.eslintrc.json
+/tsconfig.json
+
+# Lint hidden files and folders.
+!/.jest/
```
[the remark this answers is not in the document]
Comment (speedandfunction-anribolon) [reply .eslintignore:12]: I'm tryinng to set it up, but it seems eslint ignores json files. Any ideas why?

[the remark this answers is not in the document]
Comment (niedzielski) [reply .eslintignore:8]: Hm, I guess all it does is validate the JSON file. No actual formatting is done. Yeesh.

https://github.com/azeemba/eslint-plugin-json

File: package.json [hunk 3]
```diff
@@ -48,18 +54,25 @@
...
+    "eslint-config-wikimedia": "0.16.2",
+    "eslint-plugin-jest": "23.16.0",
+    "eslint-plugin-no-jquery": "2.5.0",
     "fork-ts-checker-webpack-plugin": "4.1.6",
     "jest": "26.0.0",
     "less-loader": "6.1.0",
     "mini-css-extract-plugin": "0.9.0",
     "pre-commit": "1.2.2",
     "prettier": "2.0.5",
+    "stylelint-config-wikimedia": "0.10.1",
```
Comment (niedzielski) [inline package.json:75]: Do we need stylelint itself?

File: .editorconfig [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+[*]
+indent_style = tab
+indent_size = 1
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[{package.json,*.yml}]
```
Comment (niedzielski) [inline .editorconfig:11]: We don't have any YAML files as far as I know.

File: .editorconfig [hunk 5]
```diff
@@ -0,0 +1,13 @@
+root = true
```
Comment (niedzielski) [inline .editorconfig:1]: Would you mind moving this .editorconfig to a new PR?

Comment (speedandfunction-anribolon) [reply .editorconfig:1]: ok

**Pull request merged**